app-text/multimarkdown/multimarkdown-6.4.0.recipe

```diff
@@ -0,0 +1,84 @@
+SUMMARY="Lightweight markup processor to produce HTML, LaTeX, and more"
+DESCRIPTION="MultiMarkdown, or MMD, is a tool to help turn minimally marked-up \
+plain text into well formatted documents, including HTML, PDF (by way of LaTeX), \
+OPML, or OpenDocument (specifically, Flat OpenDocument or ‘.fodt’, which can \
+in turn be converted into RTF, Microsoft Word, or virtually any other \
+word-processing format)."
+HOMEPAGE="https://fletcherpenney.net/multimarkdown/"
+COPYRIGHT="2016-2018, Fletcher T. Penney"
+LICENSE="multimarkdown"
+REVISION="1"
+SOURCE_URI="https://github.com/fletcher/MultiMarkdown-6/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="fa9daa089bc0f8bc1f69fd7365778d59210b446ce7008d03b87fb572df8ff5d3"
+SOURCE_DIR="MultiMarkdown-6-$portVersion"
+PATCHES="multimarkdown-$portVersion.patch"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+		multimarkdown$secondaryArchSuffix = $portVersion
+		cmd:markdown$secondaryArchSuffix
+		cmd:mmd$secondaryArchSuffix
+		cmd:mmd2all$secondaryArchSuffix
+		cmd:mmd2epub$secondaryArchSuffix
+		cmd:mmd2fodt$secondaryArchSuffix
+		cmd:mmd2odt$secondaryArchSuffix
+		cmd:mmd2opml$secondaryArchSuffix
+		cmd:mmd2pdf$secondaryArchSuffix
+		cmd:mmd2tex$secondaryArchSuffix
+		cmd:multimarkdown$secondaryArchSuffix
+		"
+
```

Comment:
No empty line here see the guidelines https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines